designs/administrator/olm/list-of-operators/list-of-operators.md

```diff
@@ -0,0 +1,51 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All projects view
+This is the “all-projects” view which has been modified to now expose more information about the namespaces and without listing out every CSV. The goal of this view is to consolidate the list of operators. There are a few additions to the flow:
+
+1. We have introduced the Operator Namespace column. The Operator Namespace column has 2 descriptors that can live in each row; the first is a single project like “my-project” that is associated with an operator like 3scale. If a namespace appears across All Namespaces it will appear as All Namespaces. In the example of an operator like Jaeger, the text underneath has the namespace name and All Namespaces underneath.
```

Comment:
@rachael-phillips Do we want a field level help icon next to the column headers for "operator namespace" and "managed namespace" so users can get clarification on the difference? Thinking popover explanation could be helpful here if someone needed it.

Reply:
Yes, absolutely.

Reply:
Hi @chart, @itsptk, and @tlwu2013 here is the drafted tooltip copy. Give me a +1 if this is good to go. 

Tooltip on Operator Namespace: "The Namespace(s) where the operator is installed."
Tooltip on Managed Namespace: "The Namespace(s) that the operator is watching."

Reply:
There will always just be one Operator Namespace so that can be singular, other wise sounds good to me. 👍